crates/recent_projects/src/recent_projects.rs

```diff
@@ -10,16 +10,20 @@ use picker::{
 };
 use serde::Deserialize;
 use std::{path::Path, sync::Arc};
-use ui::{prelude::*, tooltip_container, ListItem, ListItemSpacing, Tooltip};
+use ui::{prelude::*, tooltip_container, Divider, ListItem, ListItemSpacing, Tooltip};
 use util::paths::PathExt;
 use workspace::{ModalView, Workspace, WorkspaceId, WorkspaceLocation, WORKSPACE_DB};
 
 #[derive(PartialEq, Clone, Deserialize, Default)]
 pub struct OpenRecent {
-    #[serde(default)]
+    #[serde(default = "default_create_new_window")]
```

Comment:
The reasons this is not the default were described here: https://github.com/zed-industries/zed/issues/8651#issuecomment-1973390500

It is flexible enough already so that people can change their configs and make a new default.
What is the reason to change it?